keel-sql/src/main/resources/db/changelog/20200302-drop-api-version.yml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+- changeSet:
+    id: drop-api-version
+    author: fletch
+    changes:
+    - sql:
+        sql: >
+          update resource
+          set kind = concat_ws(
+            '/',
+            substring(api_version, 1, locate('.', api_version) - 1),
```

Comment:
If you had `apiVersion = 'ec2.spinnaker.netflix.com/v1'` and `kind = 'cluster'`, this would translate to `ec2/v1/cluster`, right? Again, I wasn't clear on whether we want to drop the rest of the API name...

Reply:
Correct